tests/test_openapi.py

```diff
@@ -34,6 +33,7 @@
     OperationHandler,
 )
 from inmanta.protocol.openapi.model import MediaType, Parameter, Schema
+from openapi_spec_validator import openapi_v3_spec_validator
```

Comment:
shouldn't this be in the third party section?

Reply:
yes, added openapi_spec_validator to 3rd party, resolved